src/pykx/pandas_api/pandas_meta.py

```diff
@@ -210,6 +244,37 @@ def abs(self, numeric_only=False):
             tab = _get_numeric_only_subtable(self)
         return q.abs(tab)
 
+    @api_return
+    def round(self, decimals: Union[int, Dict[str, int]] = 0):
+        tab = self
+        if 'Keyed' in str(type(tab)):
+            tab = q('{(keys x) _ 0!x}', tab)
+
+        affected_cols = _get_float_or_real_cols(tab)
+        type_dict = dict([(c, _typenum_to_typechar_mapping[tab[c].t].upper())
+                          for c in affected_cols])
+
+        # receives a column to decimals dictionary (d) and a column to
+        # typechar dictionary (t) and generates the functional qSQL operations
+        # i.e. {y$.Q.f[z]x}[;"F";2]' `c1
+        generate_ops = q(
+            "{[d;t]"
+            "({[c;d;t]"
+            "round:{string[y][0]$.Q.f[z]x};"
+            "((round[;t[c];d[c]]');c)}"
+            "[;d;t]')key[d]}")
```

Comment:
This is more about code style, I prefer to keep the function variables (d and t) in the same order in all functions, like this:
```
generate_ops = q(
  "{[d;t]"
  "({[d;t;c]"
  "round:{string[y][0]$.Q.f[x]z};"
  "((round[d[c];t[c];]');c)}"
  "[d;t;]')key[d]}")
```
What do you think?

Reply:
Not relevant with the new implementation